src/schema/message.js

```diff
@@ -11,7 +11,32 @@ export const messageSchema = new Schema({
     code_block: schema.spec.nodes.get('code_block'),
     text: schema.spec.nodes.get('text'),
     hard_break: schema.spec.nodes.get('hard_break'),
-    image: schema.spec.nodes.get('image'),
+    image: {
+      ...schema.spec.nodes.get('image'),
+      attrs: {
+        ...schema.spec.nodes.get('image').attrs,
+        height: {default: null}
+      },
+      parseDOM: [{
+        tag: 'img[src]',
+        getAttrs: dom => ({
+          src: dom.getAttribute('src'),
+          title: dom.getAttribute('title'),
+          alt: dom.getAttribute('alt'),
+          height: parseInt(dom.style.height, 10)
+        })
+      }],
+      toDOM: node => {
+        const attrs = {
+          src: node.attrs.src,
+          alt: node.attrs.alt
```

Comment:
Let the height attribute stay here anyway, its helps the browser to determine the layout

Reply:
Done